spec/models/food_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
```

Comment:
You have failed tests. Kindly fix them as well.

<img width="582" alt="Screen Shot 2022-10-16 at 06 34 39" src="https://user-images.githubusercontent.com/69011963/196013861-8d3753f4-b665-41b0-8807-f629ef99a5ec.png">